contracts/Strategy.sol

```diff
@@ -80,6 +80,15 @@ contract Strategy is BaseStrategy {
         targetRatioMultiplier = _targetRatioMultiplier;
     }
 
+    function migrateSusdVault(IVault _sUSDVault) external onlyGovernance {
+        uint256 _balanceToWithdraw = balanceOfSusdInVault();
+        withdrawFromSUSDVault(_balanceToWithdraw);
+        IERC20(susd).safeApprove(address(susdVault), 0);
+        susdVault = _sUSDVault;
+        IERC20(susd).safeApprove(address(_sUSDVault), type(uint256).max);
+        _sUSDVault.deposit();
```

Comment:
perhaps in this case we should use `newSusdVault` instead of `_sUSDVault` to avoid confusion. up to you.

Reply:
grrrrrr, newSusdVault sounds cool, tbh I used this notation for you...